test/functional/dataElements/xdmObject.spec.js

```diff
@@ -103,6 +105,26 @@ test("initializes form fields with individual object attribute values", async ()
   await stringEdit.expectValue("Adobe");
 });
 
+test.only("ensures invalid token error", async () => {
+  // temporarily remove sandboxes mock
+  await t.removeRequestHooks(platformMocks.sandboxes);
+  // replace with unauthorized mock
+  await t.addRequestHooks(platformMocks.unauthorized);
+  await initializeExtensionView();
+  await genericAlert.expectTitle("Your access token appears to be invalid.");
```

Comment:
Hmm...I don't think this test is actually working. What made me suspicious is that the error message should show up in the body of the alert and not the header. When I try to run the test on my machine, the test passes before the UI even loads, which shouldn't happen. I can also change the expected message and it still passes as well. Let me investigate this.